compose/ui/ui-text/src/skikoMain/kotlin/androidx/compose/ui/text/platform/PlatformFont.skiko.kt

```diff
@@ -67,13 +67,15 @@ class SystemFont(
  */
 class LoadedFont internal constructor(
     override val identity: String,
-    val data: ByteArray,
+    internal val getData: () -> ByteArray,
     override val weight: FontWeight,
     override val style: FontStyle
 ) : PlatformFont() {
     @ExperimentalTextApi
     override val loadingStrategy: FontLoadingStrategy = FontLoadingStrategy.Blocking
 
+    val data: ByteArray get() = getData()
```

Comment:
I'm not sure but maybe better to make it `by lazy` to avoid unexpected multirun for the lambda

Reply:
Thanks for suggestion! Done!